src/icesat2_tracks/local_modules/m_tools_ph3.py

```diff
@@ -1,378 +1,397 @@
-import matplotlib
-#matplotlib.use('Agg')
+
+import os
 import matplotlib.pyplot as plt
 import numpy as np
-#from scipy.io import netcdf
-#from mpl_toolkits.basemap import Basemap,cm
-import os
 import pickle
-#from matplotlib.dates import DateFormatter, MinuteLocator
-#from matplotlib import dates
-#import datetime as DT
+from matplotlib import dates
+from datetime import datetime
+import json
+import pandas as pd
+import h5py
+import warnings
+from pandas import HDFStore
+from pandas.io.pytables import PerformanceWarning
+import glob
 
 
 def dt_form_timestamp(timestamp, unit=None):
-    unit='h' if unit is None else unit
-    return (timestamp[1]-timestamp[0]).astype('m8['+unit+']')
+    unit = "h" if unit is None else unit
+    return (timestamp[1] - timestamp[0]).astype("m8[" + unit + "]")
 
 
 def tick_formatter(a, interval=2, rounder=2, expt_flag=True, shift=0):
 
-    O=int(np.log10(a.max()))
-    fact=10**(O-1)
-    b=np.round(a/fact, rounder+1)*fact
-    ticklabels=[' ' for i in range(len(b))]
-    N=int(np.ceil(len(b)/interval))
-
-    tt=np.arange(shift,len(b),interval)
+    O = int(np.log10(a.max()))
+    fact = 10 ** (O - 1)
+    b = np.round(a / fact, rounder + 1) * fact
+    ticklabels = [" " for i in range(len(b))]
+    
+    tt = np.arange(shift, len(b), interval)
 
     for t in tt:
         if expt_flag:
-            ticklabels[int(t)]='{:.2e}'.format(b[t])
+            ticklabels[int(t)] = "{:.2e}".format(b[t])
         else:
 
-            ticklabels[int(t)]=format(b[t], '.2f').rstrip('0').rstrip('.')#'{:.{2}f}'.format(b[t])
+            ticklabels[int(t)] = format(b[t], ".2f").rstrip("0").rstrip(".")
 
-    #ticks=a
     return ticklabels, b
 
 
 def freq_lim_string(low, high):
-	a='%2.1e' % low
-	b='%2.1e' % high
+    a = "%2.1e" % low
+    b = "%2.1e" % high
```

Comment:
Old style formatting.
```suggestion
    a = f"{low:.1e}"
    b = f"{high:.1e}"
```

Reply:
file re-formatted